### ISSUE TITLE
document selectedItemComponent option

### ISSUE THREAD

**Issue by dishcandanty** [cibernox/ember-power-select#1226]
I think this is just me not seeing this in the documentation/issues anywhere...

But I ended up digging through the addon/modules trying to find out what property was being passed to selectedItemComponent.

An example of the selectedItemComponent component as well as some documentation would certainly be helpful to new / less competent users (yup, that's me :) ).

**Comment by cibernox** [comment]
You're right, it should be documented. Probably the best place would be https://ember-power-select.com/docs/the-trigger.

Could you document there that it receives `select`, `extra` and `option`?
